### PR TITLE
Fix tox splitting test dirs, run on PR

### PR DESCRIPTION
**Pull request by samuel-w**
Fixes test splitting. Also the excludes are applying now.

https://codecov.io/gh/samuel-w/borg-import/tree/aa97f54dde531ecdbfaf2677dd7495ebf4014f3f

### CODE REVIEW

**Comment by ThomasWaldmann** [comment]
We definitely don't want to get rid of tox (tox.ini) because it is also used for local testing on the developer's machine.

**Comment by ThomasWaldmann** [comment]
Github actions now says "no jobs"?

**Comment by samuel-w** [comment]
That's because its configured to run only on push, not pul lrequest.

**Comment by samuel-w** [comment]
Example run:
https://github.com/samuel-w/borg-import/actions/runs/392971408
Example codecov:
https://codecov.io/github/samuel-w/borg-import/commit/aa97f54dde531ecdbfaf2677dd7495ebf4014f3f

File: .coveragerc [hunk 1]
```diff
@@ -1,6 +1,5 @@
 [run]
 branch = True
-source = borg
```
Comment (ThomasWaldmann) [inline .coveragerc:3]: shouldn't that be `borg_import` or `src/borg_import` ?

**Review by ThomasWaldmann — COMMENTED** [review]
hmm, one general thought:

tox is often used as a runner for all kinds of things (not just pytest, but also linters and other stuff).

if we don't use tox on the gh workflow, we basically have to implement stuff twice: once for gh, once for tox / local.

as the motivation for this change was just coverage processing (right?), the question comes up whether this can not also be solved with tox somehow?

**Comment by samuel-w** [comment]
Fixed so it runs on PR and still uses tox.

File: .coveragerc [hunk 2]
```diff
@@ -1,8 +1,9 @@
...
+source = src/borg-import
```
Comment (ThomasWaldmann) [inline .coveragerc:3]: src/borg_import

File: .coveragerc [hunk 3]
```diff
@@ -1,8 +1,9 @@
...
 omit =
     */borg_import/helpers/testsuite/*
+relative_files = True
```
Comment (ThomasWaldmann) [inline .coveragerc:6]: interesting, didn't know that one yet.

**Review by ThomasWaldmann — COMMENTED** [review]
guess this change could be interesting for some other repos also.

coveragepy somehow was problematic since longer (ever?)...

**Comment by ThomasWaldmann** [comment]
thanks!

**Pull request merged**